tedana/decomposition/ma_pca.py

```diff
@@ -476,9 +479,12 @@ def ma_pca(data_nib, mask_nib, criteria='mdl'):
     [Nx, Ny, Nz, Nt] = data_nib.shape
     data_nib_V = np.reshape(data_nib, (Nx * Ny * Nz, Nt), order='F')
     maskvec = np.reshape(mask_nib, Nx * Ny * Nz, order='F')
-    data_non_normalized = data_nib_V[maskvec == 1, :]
+    data = data_nib_V[maskvec == 1, :]
     scaler = StandardScaler(with_mean=True, with_std=True)
```

Comment:
Since the scaler is only used if `normalize` is True, could we move it into the body of the `if` statement?